sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DDLParser.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.sql.types._
 /**
  * A parser for foreign DDL commands.
  */
-class DDLParser(fallback: => ParserDialect)
+class DDLParser(fallback: => ParserInterface)
```

Comment:
Yeah I missed that one. ```AbstractSparkSQLParser``` already implements this, it should have been overridden.